includes/definitions/mobileiron.yaml

```diff
@@ -0,0 +1,33 @@
+os: mobileiron
```

Comment:
is mobleiron the best name for this OS?  it seems to just be "Core"

Reply:
The vendor doesn't give it an explicit name. MobileIron is the product line (and vendor) and Core/Sentry are the two types of appliance (core being UI/control plane, sentry being the edge).

Other systems in the same position use the vendor/product name, trendnet, raritan, opengear, etc.

I could split it into two defintions.

Reply:
It is fine, but we try to avoid vendor names if possible because it doesn't work very well when they have multiple product lines (os definitions)